docs/references/restate-sql-introspection.mdx

```diff
@@ -10,35 +10,75 @@ The schema of these tables is described in the following.
+| `service_key_utf8`   | `Utf8`   | Only contains meaningful values when a service uses a `utf8` Restate key.                                                                                                                                                                                    | `bob`                              |
+| `service_key_int32`  | `Int32`  | Only contains meaningful values when a service uses an `int32` Restate key.                                                                                                                                                                                  | `1337`                             |
+| `service_key_uuid`   | `Utf8`   | Only contains meaningful values for unkeyed Restate services.                                                                                                                                                                                                | `0189bab3b3397d828e979e2548980fa6` |
+| `service_key_json`   | `Utf8`   | Contains a JSON representation of the service key, which is a useful default if you're not sure of its shape.                                                                                                                                                | `{"name": "bob"}`                  |
+| `status`             | `Utf8`   | Enum value: `invoked` or `suspended`.                                                                                                                                                                                                                        |                                    |
+| `id`                 | `Utf8`   | [Invocation ID](/services/invocation#invocation-identifier), an opaque token that can be used to cancel an invocation of a service with the [admin API](/references/admin-api).                                                                              |                                    |
+| `invoked_by`         | `Utf8`   | Enum describing the invoker of this service: `ingress` = invoked externally / `service` = invoked by a service.                                                                                                                                              |                                    |
+| `invoked_by_service` | `Utf8`   | The name of the invoking service. Or `null` if invoked via the ingress.                                                                                                                                                                                      | `com.example.Greeter`              |
+| `invoked_by_id`      | `Utf8`   | The Invocation ID of the service that triggered this invocation. Or `null` if invoked externally.                                                                                                                                                            |                                    |
```

Comment:
`pinned_endpoint_id` seems to be missing.

Reply:
can you refresh? i added this already